### ISSUE TITLE
settings.ROOT must be defined for pdf/csv export and there is no clear error message when it is not

### ISSUE THREAD

**Issue by Gilbishkosma** [Pierre-Sassoulas/django-survey#27]
Hii,i m giving the path of survey in ROOT under settings.py but it takes different path for both csv and pdf. and i can't get how to write correct ROOT for both and for tex also.

**Comment by Pierre-Sassoulas** [comment]
Hello, thank you for using django-survey and thank you for your report.

If you changed the default path you might have to create a `tex`or a `csv`directory in your new path directory.

If this is not working, can you give your error message ?

**Comment by Gilbishkosma** [comment]
The Real error is "Settings" object has no attribute 'ROOT'.

**Comment by Gilbishkosma** [comment]
I am trying to write the path of csv and tex in ROOT on my own. But it ain't working, what should i write in ROOT?

**Comment by Pierre-Sassoulas** [comment]
You found a bug, I'll make some change so it does not happen again when I can find the time.

In the meantime, add the following line in your `settings.py` : 

```
import os
ROOT = os.path.dirname(os.path.abspath(__file__))
```

If your `settings.py` is at the same level than the `csv` and `tex`directory, it will works. You might want to define the path another way for example if your settings is in `settings/settings.py` :

`ROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "settings/")`
or
`ROOT = "/home/Gilbishkosma/workspace/django-survey/settings/`

I hope this help.

**Comment by Gilbishkosma** [comment]
Thanks Pierre-Sassoulas. The second method worked for me

**Comment by vflanker** [comment]
@Gilbishkosma @Pierre-Sassoulas I got the same error too. But I could not simply set ROOT to os.path.dirname(os.path.abspath(__file__)) cause it won't work. I set ROOT to "'D:\Python36\Lib\site-packages" but there is not tex folder, and it brings me another error:OSError: Must fix D:\Python36\Lib\site-packages\tex in order to generate tex : [Errno 2] No such file or directory: 'D:\\Python36\\Lib\\site-packages\\tex\\.tex'
I use python3.6.7 in x64 with django 2.1. tex version is 1.8 installed via "pip install tex".
I'm looking forward to your reply. Thank you.

**Comment by Pierre-Sassoulas** [comment]
@vflanker the "tex" directory should be created if you're not using the default one. It's not related to the tex package, it's something specific to django-survey that I should probably have named "django_survey_tex_output" .

**Comment by Pierre-Sassoulas** [comment]
Closed in favour of #74
